pkgs/by-name/po/podman-desktop/package.nix

```diff
@@ -148,7 +161,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [
       booxter
-      panda2134
```

Comment:
Why removing this maintainer ?

Reply:
Because the maintainer is not responsive. (It's a backport and I included it while going through the list of patches to the package to backport - since we have to do the backport for security reasons anyway.)